battery_management_systems/lg_resu_prime_http/manifest.yml

```
@@ -0,0 +1,102 @@
+blueprint_spec: device/1.0
+display_name: LG RESU Prime
+icon: enapter-battery-storage
+
+communication_modules:
+  eth:
+    product: ENP-VIRTUAL
+    lua_file: firmware.lua
+
+properties:
+  model:
+    display_name: Device Model
+    type: string
+    enum:
+      - LG RESU10 Prime
+      - LG RESU16 Prime
+  ip_address:
+    display_name: IP Address
+    type: string
+
+telemetry:
+  battery_soc:
+    display_name: State of Charge
+    description: Current State of Charge
+    type: float
+    unit: percent
+  battery_energy:
+    display_name: Battery Energy
+    description: Amount Energy of the Battery
+    type: float
+    unit: watt
+  battery_voltage:
+    display_name: Battery Voltage
+    type: float
+    unit: volt
+  battery_power:
+    display_name: Power Flow
+    description: Amount of power flowing in or out of the battery
+    type: float
+    unit: watt
+  battery_temp:
+    display_name: Battery Temperature
+    description: Current temperature of the battery
+    type: float
+    unit: celsius
+  status:
+    display_name: Battery status
+    type: string
+    enum:
+      - 'Off'
+      - Standby
+      - Initializing
+      - Charging
+      - Discharging
+      - Fault
+      - Idle
+  lastresponse:
+    display_name: Last Response
+    description: Last Response from battery
+    type: string
+
+alerts:
+  lowbattery:
+    severity: error
+    code: E001
+    display_name: Low battery
+    description: Battery has less then 10% load
+
+command_groups:
+  connection:
+    display_name: Connection
+
+commands:
+  # Connection Config
+  write_configuration:
+    populate_values_command: read_configuration
+    display_name: Configure Connection
+    group: connection
+    ui:
+      icon: file-settings-outline
+    arguments:
+      ip_address:
+        display_name: IP Address
+        description: IP address of the LG Prime Battery
+        type: string
+        required: true
+
```

Without `read_configuration` in manifest population (line `76`) won't work.

```suggestion
 read_configuration:
    display_name: Read Connection Config
    group: connection
    ui:
      icon: file-settings-outline
      
```

---

If you want user to see the populated value, then answer is no; you can leave only `write_configuration` without populating IP address.
In my opinion you could leave `read_configuration` command, renaming it like:
`display_name: Show IP address` or `display_name: Read IP address`. It seems less confusing to me.

---

in the future we'll implement a possibility to mark some commands as unlisted (hide them in UI), for now I suggest to return `read_configuration` command (maybe renamed) to make values population to work.

---

I returned the function but moved it to a group "Internals". 
Maybe this whole group can be hidden from the UX. I will file an issue here

---

Ok, thank you!